README.md

```diff
@@ -59,7 +62,7 @@ $ dispatch exec hello-py --wait
 ### Create the Event Driver
 Next we need to create the event driver. This requires two steps. First we need to create the event driver type. We can do that with the following command.
 ```bash
-$ dispatch create eventdrivertype cron dispatchframework/cron-driver:0.0.1
```

Comment:
It's fine with me to change this, but the image hasn't been pushed to this location (that I'm aware of)

Reply:
I did push a dispatchframework/dispatch-events-cron:0.0.1 on Tuesday. I pushed :latest just now.